src/Lime.Messaging/Contents/Reply.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Runtime.Serialization;
+using Lime.Protocol;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Lime.Messaging.Contents
+{
+    /// <summary>
+    /// Represents a reply message.
+    /// </summary>
+    [DataContract]
+    public class Reply : Document
+    {
+        public static readonly string MimeType = "application/vnd.lime.reply+json";
+        public static readonly MediaType MediaType = MediaType.Parse(MimeType);
+
+        public const string CONTENT = "content";
+        public const string REPLY_TO = "replyTo";
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Reply"/> class.
+        /// </summary>
+        public Reply()
+            : base(MediaType)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the content of the reply.
+        /// </summary>
+        [DataMember(Name = CONTENT)]
+        public DocumentContainer Content { get; set; }
+
+        /// <summary>
+        /// Gets or sets the reference to the message being replied to.
+        /// </summary>
+        [DataMember(Name = REPLY_TO)]
+        public ReplyTo ReplyTo { get; set; }
```

Comment:
Suggestion: change this property name to something less 'imperative'.
"ReplyTo" gives me the idea that I MUST reply using this data (like the e-mail header...)

"InReplyTo" gives the right meaning to this content, in my opinion.